packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -138,12 +139,18 @@ class InspectorTreeControllerFlutter extends Object
     }
     return _maxIndent;
   }
+
+  void setFocus() {
```

Comment:
rename to requestFocus to match Flutter conventions and avoid methods that start with the word `set`

Reply:
Done